services/httpd/listen_test.go

```diff
@@ -82,15 +82,22 @@ func BenchmarkLimitListener(b *testing.B) {
 	wg.Add(b.N)
 
 	l := httpd.LimitListener(&fakeListener{}, b.N)
+	errC := make(chan error, 1)
```

Comment:
Same comment as above: unbuffered channel, goroutine that waits on the WaitGroup and closes the channel, then use `range` and `t.Error()` instead of `Fatal`